src/web/app/components/question-responses/per-question-view-responses/per-question-view-responses.component.ts

```diff
@@ -89,7 +89,14 @@ export class PerQuestionViewResponsesComponent extends InstructorResponsesViewBa
 
       responsesToShow.push(response);
     }
-    this.responsesToShow = responsesToShow;
+
+    const hasNonMissingResponse: boolean =
+        !!responsesToShow.filter((response: ResponseOutput) => !response.isMissingResponse).length;
+    if (hasNonMissingResponse) {
+      this.responsesToShow = responsesToShow;
+    } else {
+      this.responsesToShow = [];
```

Comment:
So we are saying if all responses are missing responses then we are going to show it as no response? Looks confusing under the case where `indicateMissingResponses` is true.

Reply:
Knowing which responses are missing is not meaningful when there are no actual responses that act as the control.